presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java

```diff
@@ -61,4 +64,30 @@ public void testOrderByChar()
 
         assertEquals(actual, expected);
     }
+
+    @Test
+    public void testRcTextCharDecoding()
+            throws Exception
+    {
+        testRcTextCharDecoding(false, false);
```

Comment:
Having test method with to boolean parameter isn't too intutive, maybe we can use some annotation magic or something here?